Source/WebCore/SourcesCocoa.txt

```diff
@@ -197,6 +197,8 @@ editing/cocoa/FontAttributeChangesCocoa.mm
 editing/cocoa/FontAttributesCocoa.mm
 editing/cocoa/FontShadowCocoa.mm
 editing/cocoa/HTMLConverter.mm @no-unify
+editing/cocoa/NodeHTMLConverter.mm @no-unify
+editing/cocoa/EditingHTMLConverter.mm @no-unify
```

Comment:
Why @no-unify? 

Honestly, I am not clear on what the criteria for anything being @no-unify is, so I am quite curious.

Reply:
I am not sure precedent without rationale is a good reason here. 

Having "syntax highlighting and code completion when using Xcode" working only in some files seems so lame. We really got either fix or remove the unified build :\.

Reply:
Ya, I agree with that. In the meantime though I think I'll leave things as-is since this isn't really changing anything from the status quo. (and that can be done in a separate change if desired)